data/yam/agama/auto/template.libsonnet

```diff
@@ -31,8 +32,12 @@ function(bootloader=true,
          ssl_certificates=false,
          storage='',
          user=true) {
-  [if bootloader then 'bootloader']: base_lib.stop_timeout(),
-  [if bootloader_timeout then 'bootloader']: base_lib.timeout(),
+  [if bootloader || bootloader_timeout !='' ||
+    bootloader_extra_kernel_params != '' then 'bootloader']: std.prune({
+    stopOnBootMenu: if bootloader then true,
+    timeout: if bootloader_timeout then 15,
+    extraKernelParams: if bootloader_extra_kernel_params != '' then bootloader_extra_kernel_params,
+  }),
```

Comment:
```suggestion
[if bootloader || bootloader_timeout || bootloader_extra_kernel_params != '' then 'bootloader']: base_lib.bootloader(bootloader, bootloader_timeout, bootloader_extra_kernel_params),
```

Reply:
another improvement would be if it would be possible to do
`[if bootloader || bootloader_timeout || bootloader_extra_kernel_params != '' then 'bootloader']:` can go inside the function, so here we only run `base_lib.bootloader(..)`

Reply:
updated, thanks.